test/uk/gov/hmrc/pillar2stubs/controllers/RegisterWithoutIdControllerSpec.scala

```diff
@@ -20,11 +20,11 @@ import org.scalatest.OptionValues
 import org.scalatest.freespec.AnyFreeSpec
 import org.scalatest.matchers.should.Matchers
 import org.scalatestplus.play.guice.GuiceOneAppPerSuite
-import play.api.http.Status._
+import play.api.http.Status
 import play.api.libs.json.{JsValue, Json}
 import play.api.mvc.Result
 import play.api.test.FakeRequest
-import play.api.test.Helpers.{POST, contentAsString, defaultAwaitTimeout, route, status}
+import play.api.test.Helpers._
```

Comment:
Please address the previous comment on this: https://github.com/hmrc/pillar2-stubs/pull/100/files#r2477209013 or revert to the 5 explicit imports now that the scalafix config has been reverted.